appveyor.yml

```diff
@@ -30,3 +22,9 @@ test_script:
   - sbt ++2.11.11 amm/test
   - sbt ++2.11.11 integration/test
 
+cache:
+  - C:\sbt\
+  - C:\Users\appveyor\.m2
```

Comment:
@alexarchambault any recommendations for coursier cache on windows ? As there is a chance I'm using incorrect dir here.